acceptance-tests/src/test/java/tech/pegasys/signers/secp256k1/tests/multikey/signing/MultiKeyTransactionSigningAcceptanceTestBase.java

```diff
@@ -33,13 +34,15 @@
 
 public class MultiKeyTransactionSigningAcceptanceTestBase extends MultiKeyAcceptanceTestBase {
 
-  private static final String ADDRESS = "fe3b557e8fb62b89f4916b721be55ceb828dbd73";
   private static final byte[] DATA_TO_SIGN = "42".getBytes(UTF_8);
   private static final String PRIVATE_KEY =
       "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
+  static final String PUBLIC_KEY_HEX_STRING =
+      "0x9b02f8a5fddd222ade4ea4528faefc399623af3f736be3c44f03e2df22fb792f3931a4d9573d333ca74343305762a753388c3422a86d98b713fc91c1ea04842";
```

Comment:
I noticed that "FILENAME" in other test classes start with `09b02...`, is that leading 0 intentional (i.e. Bytes' class toHexString results in that?)

Reply:
leading 0 is required ... but this tests all seem the same, but different, so may spend some time looking into it